packages/material-ui-csskit/README.md

```diff
@@ -0,0 +1,19 @@
+# @material-ui/csskit
+
+You can leverage our css helper functions that integrate with the rest of material-ui.
+
+## Installation
+
+Install the package in your project directory with:
+
+```sh
+// with npm
+npm install @material-ui/csskit
+
+// with yarn
+yarn add @material-ui/csskit
+```
+
+## Documentation
+
+So far, we implemented a helper function for generating responsive fonts, [documented here](https://material-ui.com/customization/themes#typography)
```

Comment:
It would be nice if there was an anchor attached to the documentation so we can do something like:

```suggestion
So far, we have implemented a [helper function](https://material-ui.com/customization/themes#typography) for generating responsive fonts.
```

Reply:
Agreed, will do

Reply:
You meant an anchor to the new specific doc section right? Turns out it's not there as it's only automatically generated for h1-h3 markdown headers. 

The only solutions I see is either change how the docs are generated, or moving this out of the Typography section to one of its own so it's h3, but this seemed off. But let me know if you want this.

Reply:
Added your copy. See comment above and just reopen if you disagree (re: anchor to the specific section)